[Alerting][Docs] Updating docs for notify only on action group change (#92084)
elastic/kibana: docs/user/alerting/defining-alerts.asciidoc

All alert share the following four properties in common:

[role="screenshot"]
image::images/alert-flyout-general-details.png[alt='All alerts have name, tags, check every, and notify properties in common']

Name::      The name of the alert. While this name does not have to be unique, the name can be referenced in actions and also appears in the searchable alert listing in the management UI. A distinctive name can help identify and find an alert.
Tags::      A list of tag names that can be applied to an alert. Tags can help you organize and find alerts, because tags appear in the alert listing in the management UI which is searchable by tag.
Check every::      This value determines how frequently the alert conditions below are checked. Note that the timing of background alert checks are not guaranteed, particularly for intervals of less than 10 seconds. See <<alerting-production-considerations>> for more information.
Notify::      This value limits how often actions are repeated when an alert instance remains active across alert checks. See <<alerting-concepts-suppressing-duplicate-notifications>> for more information. +

> - **Only on status change**: Actions are not repeated when an alert instance remains active across checks. Actions only run when the alert status changes.

Reviewer: ```suggestion
- **Only on status change**: Actions are not repeated when an alert instance remains active across checks. Actions run only when the alert status changes.
```

> - **Every time alert is active**: Actions are repeated each time when an alert instance remains active across checks.

Reviewer: ```suggestion
- **Every time alert is active**: Actions are repeated when an alert instance remains active across checks.
```

> - **On a custom action interval**: Actions are supressed for the throttle interval but will be repeated when an alert instance remains active across checks for a duration longer than the throttle interval.

Reviewer: ```suggestion
- **On a custom action interval**: Actions are suppressed for the throttle interval, but repeat when an alert instance remains active across checks for a duration longer than the throttle interval.
```
